powder_reduction.py

```diff
@@ -3,14 +3,35 @@
 import numpy as np
 import scipp as sc
 from scipp.neutron.diffraction import load_calibration
-import smooth_data
+from smooth_data import smooth_data
+from spline_background import bspline_background
+from absorption import absorption_correction
 
 
-def powder_reduction(sample='sample.nxs', calibration=None, lambda_binning=(0.7, 10.35, 5615)):
+def powder_reduction(sample='sample.nxs',
+                     calibration=None,
+                     vanadium=None,
+                     empty_instr=None,
+                     lambda_binning=(0.7, 10.35, 5615),
+                     dspacing_binning=(1., 10., 9000),
+                     **absorp):
```

Comment:
This implies that the same sample size is used for sample and vanadium runs. Is guess this is not true in practice, i.e., you should probably have independent absorption parameters for the two?